source/layouts/layout.html.haml

```diff
@@ -1,22 +1,24 @@
-%head
-  <!--[if IE]><script src='http://html5shiv.googlecode.com/svn/trunk/html5.js'></script><![endif]-->
-  %meta{charset:'utf-8'}
-  <meta http-equiv='X-UA-Compatible' content='IE=edge,chrome=1'>
-  %meta{content:'width=device-width', name:'viewport'}
-  %meta{name:'description', content: data.page.description}
+!!!
+%html
+  %head
+    <!--[if IE]><script src='http://html5shiv.googlecode.com/svn/trunk/html5.js'></script><![endif]-->
+    %meta{charset:'utf-8'}
+    <meta http-equiv='X-UA-Compatible' content='IE=edge,chrome=1'>
+    %meta{content:'width=device-width', name:'viewport'}
+    %meta{name:'description', content: data.page.description}
 
-  %title= current_page.data.title || settings.site_name
-  %link{href: 'feed.xml', rel: 'alternate', title: 'RSS', type: 'application/rss+xml'}
+    %title= current_page.data.title || settings.site_name
+    %link{href: 'feed.xml', rel: 'alternate', title: 'RSS', type: 'application/rss+xml'}
```

Comment:
Line is too long. [88/80]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace